docker/nginx/conf.d/server/server.account

```diff
@@ -3,6 +3,11 @@ listen 443 ssl http2;
 include /etc/nginx/conf.d/include/ssl-settings;
 include /etc/nginx/conf.d/include/init-optional-variables;
 
+# Uncomment to launch new Dashboard under /v2 path 
+# location /v2 {
+#   proxy_pass http://dashboard-v2:9000;
+# }
```

Comment:
we can uncomment it right away if that works :)

Reply:
It does, but we still need to confirm with ops if that's how we want to run it.